.github/ISSUE_TEMPLATE/crash-issues.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Crash Report
+about: For issues regarding game crashes or permanent freezes
+---
+
+<!-- After you fill in all information, delete all comments in the issue -->
+
+**What is your problem:** <!-- Name the problem you have encountered -->
+
+**Describe your problem:** <!-- Provide any information you believe could be useful -->
+
+**Screenshots or videos showing encountered issue:** 
+
+**osu!lazer version:** <!-- Provide the version of your osu!lazer, you can find it at the bottom of the screen  -->
+
+**Logs:** <!-- Attach your osu!lazer logs, you can find them under %appdata%\osu\logs in Windows, or under ~/.local/share/osu/ in Linux and macOS -->
```

Comment:
for linux/mac(from corefx code):
"$XDG_DATA_HOME defines the base directory relative to which user specific data files should be stored."
"If $XDG_DATA_HOME is either not set or empty, a default equal to $HOME/.local/share should be used.

Reply:
Linux uses `$HOME` for more consistency.

Reply:
I vote against @VPeruS's suggestion. To  be more universal, use `$HOME`. Not all Linux distributions has XDG

Reply:
That how it works, first it try $XDG_DATA_HOME if it not set it will use $HOME, hence if something presetup XDG_DATA_HOME to "/usr/share" you never find your logs